package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "20.19.3",
+  "version": "21.0.0",
```

Comment:
This should probably be `20.19.4` I think.

Reply:
Ok, I was going with what I read here...

https://github.com/department-of-veterans-affairs/vets-json-schema#:~:text=Changing%20the%20type%20on%20a%20schema%27s%20property

Reply:
@timwright12 can you shed some light on this? I just want to check for my own future reference

Reply:
I don't know enough about this data to really say whether it's a breaking change or not. @RudyOnRails would you be able to look at where it's used/implemented to confirm a full version bump?

Reply:
@timwright12 Regardless of the data, I was strictly following the readme guidelines of what was considered a major bump:
> Changing the type on a schema's property